src/network/networkd-link.c

```diff
@@ -935,7 +994,7 @@ static int address_handler(sd_netlink *rtnl, sd_netlink_message *m, void *userda
 
         if (link->address_messages == 0) {
                 log_link_debug(link, "Addresses set");
-                link_enter_set_routes(link);
+                link_enter_set_neighbors(link);
```

Comment:
Now we are setting address then routes. I think we should not touch this path. It should be done like ```address label```, ```ip routing rule``` independent of address then routes.  Any reason why  ```address->neighbor->routes``` ?

Reply:
The only reason I did it this way was to keep everything here simple. Theoretically static neighbor setting could happen in parallel with addresses and routes since there is no dependency either way on neighbors.

Reply:
I made some changes so we now have
`addresses -> routes` and `neightbors`
in parallel